### PR TITLE
Feature/visitors combine ui components

### PR DESCRIPTION
**Pull request by StasKozar**
https://trello.com/c/gufqT4uQ/59-visitors-combine-ui-components-according-to-design-and-specification

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,12 +8,26 @@ services:
       - ./backend:/app
       - ./.docker/php-fpm/php-ini-overrides.ini:/etc/php/7.3/fpm/conf.d/99-overrides.ini
 
+  frontend:
```
Comment (TatsuUkraine) [inline docker-compose.yml]: do you need this?

File: frontend/src/components/layout/Sidepanel.vue [hunk 2]
```diff
@@ -9,12 +9,15 @@
...
-            <RouterLink 
+            <RouterLink
                 v-for="link in links"
                 :key="link.text"
                 :to="{ name: link.route }"
             >
-                <VListItem class="pl-8 my-3">
+                <VListItem
+                    class="pl-8 my-3"
+                    v-if="!link.hasOwnProperty('sublinks')"
```
Comment (TatsuUkraine) [inline frontend/src/components/layout/Sidepanel.vue]: make it just `link.sublinks`

File: docker-compose.yml (hunk 1 above)
Comment (StasKozar) [reply docker-compose.yml]: Fix

File: frontend/src/components/layout/Sidepanel.vue (hunk 2 above)
Comment (StasKozar) [reply frontend/src/components/layout/Sidepanel.vue]: Fix

**Pull request merged**